src/hooks/useMassSelection.ts

```diff
@@ -26,6 +27,12 @@ export const useMassSelection = (
   );
   const [ids, setIds] = useState<string[]>([]);
 
+  useEffect(() => {
```

Comment:
We're removing ids from the list every time `idsToRemove` changes, right? I think it would be better to return a `removeIds` function from this hook that components can use to delete ids. What do you think about that?

Reply:
Looks like codecov wants more tests :/

Reply:
Just make sure your test calls `deselectIds` somewhere, and you should be good